disconnect-when-backgrounded.js

```diff
@@ -1,18 +1,30 @@
 var disconnectTimer = null;
 
+// 10 seconds by default
+var disconnectTime = (Meteor.settings && Meteor.settings.public && Meteor.settings.public.disconnectTime || 10) * 1000;
```

Comment:
i'm confused what you mean here. why would we want to recalculate the time on each disconnect as opposed to once on startup?